### ISSUE TITLE
error in sui-exec.sh

### ISSUE THREAD

**Issue by georgescharlesbrain** [ChainMovers/suibase#94]
Not sure if this is a suibase issue.  
But maybe you have a better understanding of this.

```
lsui move test
INCLUDING DEPENDENCY SuiTears
INCLUDING DEPENDENCY Sui
INCLUDING DEPENDENCY MoveStdlib
BUILDING valuencer
thread 'main' panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/stacker-0.1.15/src/lib.rs:1972024-02-28T17:58:12.055382Z ERROR telemetry_subscribers: panicked at /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/stacker-0.1.15/src/lib.rs:197:21:
setting stack permissions failed with: Cannot allocate memory (os error 12) panic.file="/home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/stacker-0.1.15/src/lib.rs" panic.line=197 panic.column=21
:21:
setting stack permissions failed with: Cannot allocate memory (os error 12)
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
/home/x/suibase/scripts/common/__sui-exec.sh: line 101: 194806 Aborted                 (core dumped) $SUI_BIN "$SUI_SUBCOMMAND" $_OPT_DEFAULT_PATH $_OPT_DEFAULT_INSTALLDIR ${CANONICAL_ARGS[@]}
```

**Comment by mario4tier** [comment]
It is a known issue. Mysten Lab is aware of it and working on a fix.

I did put on the ChainMovers discord a work-around for Suibase users.

Here is a copy:

!!! For people struggling with latest 1.19 release from Mysten Labs (because of memory allocation problem). 

Today I did enhance Suibase to support forcing the use of an older Sui binary version (do "~/suibase/update" to get this new feature). 

As an example, to force using 1.18.1 on testnet add the following line to the file "~/suibase/workdirs/testnet/suibase.yaml":
```force_tag: "testnet-v1.18.1"```

...then do a "testnet update" 

For devnet, add the following to "~/suibase/workdirs/devnet/suibase.yaml":
```force_tag: "devnet-v1.18.0"```

... then do "devnet update" (Note: there is no devnet-v1.18.1) 

Once Mysten Labs release a fix for the memory allocation issue, you just remove the force_tag from the suibase.yaml to resume getting the latest.
